ddp-workspace/projects/ddp-testboston/src/assets/i18n/en.json

```diff
@@ -142,19 +143,19 @@
                 {
                     "Question": "What will I have to do if I join the study?",
                     "Paragraphs": [
-                        "Participants in TestBoston are asked to fill out a short survey about COVID-19 and perform two simple COVID-19 sample collections on themselves at home.  Participants are asked to return the kits to us in a pre-paid box for processing. Participants are also asked to repeat this process each month for 5 additional months. The test collections are a nasal swab from the front of your nose and a finger prick blood spot."
+                        "Participants in TestBoston are asked to fill out a short survey about COVID-19 and perform two simple COVID-19 sample collections on themselves at home. Participants are asked to return the kits to us in a pre-paid box for processing. Participants will also be asked if they would like to be tested monthly for COVID-19 for the next 5 months. The test collections are a nasal swab from the front of your nose and a finger prick blood spot."
                     ]
                 },
                 {
                     "Question": "Can my family join TestBoston?",
                     "Paragraphs": [
-                        "Right now, only Brigham and Women’s patients who receive randomly-generated invitations can join TestBoston. If more than one person in your household has received an invitation letter, then you can both join the study if you choose. If you did not receive an invitation, but you are interested in staying informed of study progress, you are welcome to join the TestBoston mailing list."
+                        "Right now, only Brigham and Women’s patients who receive randomly-generated invitations can join TestBoston. If more than one person in your household has received an invitation letter, then you can both join the study if you choose. If you did not receive an invitation, but you are interested in staying informed of study progress, you are welcome to <a class=\"Link Link_semi-bold\" href=\"/#stay-informed\">join the TestBoston mailing list</a>."
```

Comment:
When I click on link, I end up at the top of the home page.

Reply:
On it